CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
-## Unreleased
-
-- [CHANGE] Disable gzip compression of kube-state-metrics responses by default. Can be re-enabled via `--enable-gzip-encoding`. See #563 for more details.
+## v1.5.0 / 2018-11-30
```

Comment:
That would be `v1.5.0-alpha.0`. We typically included all pre-releases' history in Kubernetes project releases.